### DISCUSSION TITLE
Adapting the drainage algorithm to imbibition

### DISCUSSION THREAD

**Discussion by LukasPore** [PMEAL/porespy#713] [Q&A]
Hello all,
I was wondering if there is an easy way to adapt the drainage algorithm to imbibition? Do you have any hints, or is a release planned in the future?
Best regards,
Lukas

**Comment by jgostick** [comment]
It is planned in the future.  I actually have a rough sketch of the code somewhere.  How soon and how badly do you need it?

**Reply by LukasPore** [reply]
Thank you very much for the quick reply. We would like to start working on it next month.  Maybe we can collaborate on that. I wrote you a PM to give you more context.
